### DISCUSSION TITLE
Finsh reading & rate at the same time?

### DISCUSSION THREAD

**Discussion by asmaloney** [bookwyrm-social/bookwyrm#2577] [Q&A] [answered]
I could very well be missing something, but is there a way to mark a book as finished and rate it at the same time?

I click "Finish Reading" and I expected to be able to put a rating in there. Instead it seems like I have to do that and then go back to the book page and click the stars to add a rating.

**Comment by mouse-reeve** [comment, marked as answer]
I agree! There's an issue for this here: https://github.com/bookwyrm-social/bookwyrm/issues/1160

**Reply by asmaloney** [reply]
Oh sorry - I missed that somehow!

**Reply by mouse-reeve** [reply]
No worries, it's a real bottomless pit in there :)

**Reply by asmaloney** [reply]
I wish I could work in Python. I have experience with vue.js + bulma.
